login/routers/controllers/UserExistCheck.js

```diff
@@ -4,19 +4,19 @@ const SequelizeOperators = Sequelize.Op;
 const UserModel = require('../../db/models/UserModel.js');
 const createUserAccount = require('../../db/CreateUserAccount.js');
 
-function CopyCheck(registrationRequest, response) {
+function UserExist(registrationRequest, response) {
   UserModel.User.findOne({
     raw: true,
     where: {
-      [SequelizeOperators.or]: [{ login: registrationRequest.login }, { mail: registrationRequest.mail }],
+      [SequelizeOperators.or]: [{ login: registrationRequest.login }, { email: registrationRequest.email }],
     },
   })
     .then((User) => {
       if (!User) {
         createUserAccount.createUserAccount(registrationRequest, response);
         return;
       }
-      response.send('Такой пользователь уже есть');
+      response.status(400).send('Такой пользователь уже есть');
```

Comment:
I think the proper status code for this kind of error is 409, don't just blindly trust me though, check it yourself.

Reply:
i read about 409,and briefly it is "conflict" but when i open wide-ranging description of it,there was about put requests

Reply:
Wait, isn't this a put request? You're creating the user.